src/app/shared/services/mysql.service.ts

```diff
@@ -116,11 +116,11 @@ export class MysqlService {
     });
   }
 
-  private getQueryCallback<T extends TableRow>(subscriber) {
+  private getQueryCallback<T extends TableRow>(subscriber): any {
```

Comment:
I do not like this any, it is better to put the real return type

Reply:
Fixed the data type, now I return the corresponding type.